src/python-bindings/htcondor2/submit.cpp

```diff
@@ -570,3 +568,12 @@ _submit_from_dag( PyObject *, PyObject * args ) {
 
     return PyUnicode_FromString( dag_opts[shallow::str::SubFile].c_str() );
 }
+
+static PyObject *
+_display_dag_options( PyObject *, PyObject * /*args*/ ) {
+    DagmanUtils du;
+    du.DisplayDAGManOptions("%35s   | %s\n", DagOptionSrc::PYTHON_BINDINGS, " : ");
```

Comment:
Do we really want multiple hard-coded spaces in this format string?  (If `DisplayDAGManOptions()` ignores duplicate spaces, they should be removed to avoid giving the wrong impression.)

Reply:
Discussed in person. Will leave for now, but may reconsider formatting in the future